engine/builder_wasmedge.go

```diff
@@ -4,12 +4,12 @@
 package engine
 
 import (
+	"github.com/suborbital/appspec/tenant"
 	"github.com/suborbital/sat/api"
```

Comment:
nit: newline between appspec imports and sat imports to be consistent with other changes such as 

https://github.com/suborbital/sat/pull/145/files#diff-a435ebf0c856ea9651b9c75ccbe147422877acd57cc1d9734aa71f77fb5d4f41R6-R8